ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.14-periodics.yaml

```diff
@@ -93,11 +175,12 @@ periodics:
   labels:
     ci-operator.openshift.io/cloud: equinix-ocp-metal
     ci-operator.openshift.io/cloud-cluster-profile: equinix-ocp-metal
+    ci-operator.openshift.io/cluster: build05
     ci-operator.openshift.io/variant: periodics
     ci.openshift.io/generator: prowgen
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-disconnected-conformance
+  name: periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-disconnected-ipv6-conformance
```

Comment:
Should all of these `ipv6` references to be something like dual? (if this is just the name it's fine, I just wondering that maybe we missed to move this from `ipv6` to `dual`)

Reply:
What does the `references` mean here? Here I just add `ipv6` which is using general workflow: hypershift-mce-agent-disconnected-conformance. For dualstack, i updated the `IP_STACK` inside.